src/PriceCalculatorKata/Program.cs

```diff
@@ -1,12 +1,66 @@
 ﻿using System;
-
+using System.Collections.Generic;
+using System.Globalization;
 namespace PriceCalculatorKata
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            Product product = new Product()
+            {
+                Name = "The Little Prince",
+                Price = 20.25,
+                UPC = 12345,
+                UPCDiscountPercentage = 7,
+            };
+            DiscountCalculations discountCalculations = new DiscountCalculations()
+            {
+                Product = product,
+                UniversalDiscountPercentage = 15,
+                ProductDiscountType = DiscountType.Multiplicative,
+            };
+            // discountCalculations.AssignCapAmount(ValueComputationType.Absolute, 4);
+            TaxCalculation taxCalculation = new TaxCalculation()
+            {
+                Product = product,
+                TaxPercentage = 21,
+                UniversalDiscountPrecedence = DiscountPrecedence.AfterTax,
+                UPCDiscountPrecedence = DiscountPrecedence.AfterTax,
+                UniversalDiscountAmount = discountCalculations.GetUniversalDiscountAmount(),
+                UPCDiscountAmount = discountCalculations.GetUPCDiscountAmount(),
+            };
+            AdditionalCostsCalculations additionalCostsCalculations = new AdditionalCostsCalculations()
```

Comment:
FYI, you can use 'var' like so:
var additionalCostsCalculations = new AdditionalCostsCalculatios();

No need to refactor all code for this, just an FYI ;)

Reply:
done except for 
Dictionary<string, double> additionalCosts = additionalCostsCalculations.AdditionalCosts;
so the developer could know the type returned in this line